docs/engineering/android-app/configuring/config-types/widget.mdx

```diff
@@ -160,6 +160,36 @@ primaryTextActions | The list of actions associated with the primary text. If th
 secondaryTextActions | The list of actions associated with the secondary text. If the list is empty, no actions will be displayed | | |
 overflow | Determines how the text should be truncated if it is too long to fit | | [TextOverflow.ELLIPSIS] |
 
+##How to use ACTIONABLE BUTTON
+Defines the configuration properties for a Actionable Button view, which is a view that displays button with text strings, it's status ,visibility mode, and if it is enabled. This view can also have actions associated with it.
+
+ ``` json
+ [
+   {
+     "viewType": "BUTTON",
+     "smallSized": true,
+     "text": "@{taskDescriptionWithStartDate}",
+     "status": "@{taskStatusColorCode}",
+     "visible": "true",
+     "enabled": "true",
+     "actions": []
+   }
+ ]
+```
+
+### Config properties of ACTIONABLE BUTTON
+|Property | Description | Required | Default |
+|--|--|:--:|:--:|
+|viewType|The view type should be `BUTTON` it helps in knowing the kind of view at hand|Yes| |
+|text|The text represents the button name that is what the button action name is e.g Add Patient|Yes| |
+|status| status is the current mode of the button it can be a pure text like `Started` or even the color code of different modes|Yes| |
```

Comment:
not sure if `Started` is correct, I think we are using [ServiceStatus](https://github.com/opensrp/fhircore/blob/main/android/engine/src/main/java/org/smartregister/fhircore/engine/domain/model/ServiceStatus.kt) for the type of status that the codebase support